### DISCUSSION TITLE
How can I use OneAPI?

### DISCUSSION THREAD

**Discussion by git-rohan7** [oneapi-src/oneAPI-samples#2484] [Q&A]
Please help me so that I can use OneAPI effectively.

**Comment by AniaIvanova** [comment]
For example, if you have a folder with the source code of a Fortran project (FortranProject), you can compile this project as follows. Install HPC Toolkit, run Intel oneAPI as Admin, go to the folder FortranProject. The command C:\FortranProject>ifx file1.f file2.f ... filen.f  (all files of the project) will compile it and create file1.exe (by the first file name). You can rename it, move to the folder with input file(s) and run.
